SimPEG/Regularization.py

```diff
@@ -1145,7 +1145,7 @@ class Tikhonov(BaseComboRegularization):
     def __init__(
         self, mesh,
         alpha_s=1e-6, alpha_x=1.0, alpha_y=1.0, alpha_z=1.0,
-        alpha_xx=Utils.Zero(), alpha_yy=Utils.Zero(), alpha_zz=Utils.Zero(),
+        alpha_xx=1., alpha_yy=1., alpha_zz=1.,
```

Comment:
@sgkang: I think the default for these should be 0? They could be `0.` instead of `Utils.Zero()`, but I don't think we want to be updating this default.

cc @rowanc1

Reply:
Oh, sorry this should be zero.